app/resources/css-default.template

```diff
@@ -628,7 +628,7 @@ img.icon {
 
 .view input.add-note {
   width: 100%;
-  margin-top: 10px; /* Vertically align blue buttom with record card */
+  margin-top: 10px;  /* To vertically align the "Add note" button with the record card */
```

Comment:
Yeah it would be nice if you fix them too. For the case above, it would be more readable to break line with all commas:
```css
table.api-keys .contact,
table.api-keys .contact a,
table.api-keys .contact a:visited {
...
```

It's usually more readable to break with either all commas or no commas, than breaking with some of them.